1001-2000/LIT1287Dersan.xml

```diff
@@ -90,12 +90,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
        
          </div>
          <div type="edition" xml:lang="gez">
-            <div  type="textpart" subtype="incipit"><ab>ድርሳን፡ ዘደረሰ፡ አብ፡ ብፁዕ፡ ኤጴስ፡ ቆጶስ፡ (sic) ክቡር፡ አባ፡ ቴዎዶጦስ፡ ኤጴስቆጶስ፡ ዘዕንቆራ፡ 
+            <div type="textpart" subtype="chapter" xml:id="Introduction"><label>Introduction</label>
+               
+               <div  type="textpart" subtype="incipit"><ab>ድርሳን፡ ዘደረሰ፡ አብ፡ ብፁዕ፡ ኤጴስ፡ ቆጶስ፡ (sic) ክቡር፡ አባ፡ ቴዎዶጦስ፡ ኤጴስቆጶስ፡ ዘዕንቆራ፡ 
                እምአድያመ፡ ገላትያ፡ በዕለተ፡ ተዝካሩ፡ ለፀሐየ፡ ጽድቅ፡ ክቡር፡ ወልዑል፡ ኮከበ፡ ጽባሕ፡ ብሩህ፡
                ወጽዱል፡ ወዝንቱሰ፡ አቡነ፡ ብፁዕ፡ ቴዎዶጦስ፡ ዘነበበ፡ ድርሳን፡ ገድል፡ እንተ፡ ቅዱስ፡ ማር፡ ጊዮርጊስ፡ ወውእቱኬ፡
                ፩፡ እምአበዊነ፡  ኤጴስቆጶሳት፡ ቅዱሳን፡ እለ፡ ተጋብዑ፡ በኒቅያ፡ በእንተ፡ ሃይማኖት፡ ርትዕት፡ 
```

Comment:
ኤጲስቆጶሳት?

Reply:
ኤጲስቆጶስ?

Reply:
I cannot understand where these corrections belong, sorry. this is a very very old PR

Reply:
Ok got it